src/virtual-repeat.js

```diff
@@ -271,26 +271,43 @@ export class VirtualRepeat extends AbstractRepeater {
   _getMore(): void {
     if (this.isLastIndex || this._first === 0) {
       if (!this._calledGetMore) {
-        let getMoreFunc = this.view(0).firstChild.getAttribute('virtual-repeat-next');
-        if (!getMoreFunc) {
-          return;
-        }
-        let getMore = this.scope.overrideContext.bindingContext[getMoreFunc];
         let executeGetMore = () => {
           this._calledGetMore = true;
-          if (getMore instanceof Promise) {
-            return getMore.then(() => {
-              this._calledGetMore = false; //Reset for the next time
-            });
-          } else if (typeof getMore === 'function') {
-            let result = getMore.bind(this.scope.overrideContext.bindingContext)(this._first, this._bottomBufferHeight === 0, this._isAtTop);
-            if (!(result instanceof Promise)) {
-              this._calledGetMore = false; //Reset for the next time
-            } else {
-              return result.then(() => {
+          let func = this.view(0) ? this.view(0).firstChild.au['virtual-repeat-next'].instruction.attributes['virtual-repeat-next'] : undefined;
```

Comment:
@AStoker When running `virtual-repeat` without the `virtual-repeat-next` attribute. I think `this.view(0).firstChild.au` needs to be null checked as well.